contrib/pipelines/secadvisor/Makefile

```diff
@@ -6,7 +6,7 @@ SRCS := \
 
 .PHONY: all
 all:
-	go build -o $(PROG) $(SRCS)
+	go build -o $(PROG) -ldflags '-extldflags "-static"' $(SRCS)
```

Comment:
It is not common and/or required to force static compile in 'all' please move to 'static'. Also this is aligned with Skydive compile which is not-static by defaule and static with running  make static

Reply:
ok, will add target 'static'